### PR TITLE
Feature/push notification

### PR DESCRIPTION
**Pull request by Rodrive**

### CODE REVIEW

File: routes/rides.mjs [hunk 1]
```diff
@@ -61,6 +65,19 @@ const router = generateCRUD(Ride, {
...
       ]);
+      NotificationDevice.findOneByUser(ride.driver.id).then((device) => {
+        if (device) {
+          const payload = {
+            message: `Nouvelle course à ${DateTime.fromJSDate(ride.start).setLocale('fr').toFormat('HH\'h\'mm')}`,
+            body: `De ${ride.departure.label} à ${ride.arrival.label}`,
+          };
+          device.notify(payload).catch((e) => {
+            if (e.name === 'WebPushError') {
+              NotificationDevice.deleteOne({ _id: device._id.toString() }).then();
```
Comment (Nainterceptor) [inline routes/rides.mjs]: final .then ?
Is it to ensure that something is executed ? In this case I think .exec() will be available here

File: routes/rides.mjs [hunk 2]
```diff
@@ -61,6 +65,19 @@ const router = generateCRUD(Ride, {
         `campus/${ride.campus.id}`,
         `driver/${ride.driver.id}`,
       ]);
+      NotificationDevice.findOneByUser(ride.driver.id).then((device) => {
```
Comment (Nainterceptor) [inline routes/rides.mjs]: Is it possible to use async/await ?

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -74,6 +74,8 @@ deploy_review:
       --set env.MAIL__CONTACT_MAIL="${STAGING_MAIL__CONTACT_MAIL}"
       --set env.REDIRECT_TARGET="${STAGING_REDIRECT_TARGET}"
       --set env.USER_WEBSITE_URL="${STAGING_USER_WEBSITE_URL}"
+      --set env.VAPID__PUBLIC_KEY="${STAGING_VAPID__PUBLIC_KEY}"
```
Comment (Nainterceptor) [inline .gitlab-ci.yml]: I've not seen corresponding key in 
* `api/.helm-chart/values.yaml` (after l.15)
* `api/.helm-chart/templates/deployment.yaml`

File: routes/rides.mjs (hunk 1 above)
Comment (Rodrive) [reply routes/rides.mjs]: This is to not block the answer. Added a //todo to add this to a queue system

File: routes/rides.mjs (hunk 2 above)
Comment (Rodrive) [reply routes/rides.mjs]: This is to not block the answer. Added a //todo to add this to a queue system

**Comment by Rodrive** [comment]
[Notifications Chauffeurs quand telephone en veille](https://trello.com/c/JgBc2YZb/261-notifications-chauffeurs-quand-telephone-en-veille)

**Pull request merged**